nixos/lib/testing-python.nix

```diff
@@ -87,10 +90,101 @@ rec {
         inherit driver;
       };
 
-      inherit pos;
+      inherit pos; # for better debugging
     };
 
+  # Generate convenience wrappers for running the test driver
+  # has vlans, vms and test script defaulted through env variables
+  # also instantiates test script with nodes, if it's a function (contract)
+  mkDriver = {
```

Comment:
`mkTestDriver` and `mkDriver` are somehow not the most self-describing names.

i think the `mkTestDriver` attribute could just be `pythonTestDriver` or so (because it's just the naked python script wrapped with some generic runtime env info).
`mkDriver` could be `setupDriverForTest` or something like that (because we're taking the script and basically add all the needed parameters that are finally test-specific).